examples/ecs/fallible_systems.rs

```diff
@@ -6,13 +6,33 @@ use bevy::prelude::*;
 use rand::distributions::Distribution;
 
 fn main() {
-    App::new()
-        .add_plugins(DefaultPlugins)
-        .add_systems(Startup, setup)
-        .run();
+    let mut app = App::new();
+
+    app.add_plugins(DefaultPlugins)
+        .add_systems(Startup, (setup, failing_system))
+        .add_systems(PostStartup, failing_system)
+        // You can set a global error handler for fallible systems.
+        //
+        // By default, fallible systems that return an error will panic.
+        .set_systems_error_handler(|err, system| warn!("{:?} failed: {err}", system.name()))
+        // You can handle individual systems by piping the output result.
+        .add_systems(
+            PostStartup,
+            failing_system.pipe(|result: In<Result>| {
+                result.0.inspect_err(|err| info!("captured error: {err}"))
+            }),
+        );
+
+    // You can also set a custom error handler per `Schedule` (optionally at runtime through
```

Comment:
Phrasing here could be cleaned up. I think the parenthetical can just be removed TBH.

Reply:
Resolved in [95b54ba](https://github.com/bevyengine/bevy/pull/17753/commits/95b54bac8e63257562d3e5d6ef805455233e17bc).